ch4/exercise4-9/wordfreq.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	
+)
+
+func main() {
+	wordfreq("wordfreq.txt")
+}
+
+func wordfreq(file string) {
+	wordCounts := make(map[string]int)
+    f, _ := os.Open(file)
```

Comment:
Nit: indentation.  Are you sure you're running everything through `goftmt`/`goimports`?  Stuff like this should be auto handled if that is set up correctly.

Reply:
I don't think it is set up correctly, I will set it up.